File: day-5/is_sort.rb
def is_sort(arr)
  length = arr.length
  puts length

  if length <= 1
    return true
  end

  (0...arr.length - 1).each do |i|
    # Compare current element with the NEXT element
    if arr[i] > arr[i + 1]
      return false
    end
  end

  return true
end


puts is_sort([1, 4])
puts is_sort([2, 7, 9, 10])

File: day-5/isSort.py
def isSort(arr):
    length_arr = len(arr)
    if length_arr == 1:
        return True
    for i in range (length_arr-1):
        if arr[i] > arr[i + 1]:
            return False
        
    return True


print(isSort([1]))
print(isSort([2, 1, 7, 3, 9]))
print(isSort([ 1, 3, 9, 11]))